src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Colors randomColor = new ColorSupplier().getRandomColor();
+        int randomNumber = new Random().nextInt(100);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/156338484/1bae2894-4f3a-41b1-8a8c-40a0767a5258)

The same

Reply:
All issues fixed